asset-registration/README.md

```diff
@@ -0,0 +1,62 @@
+# Model and Dataset Registration
+
+## Introduction
+
+Each team developing a model works in isolation in their own TRE workspace, with their own AML instance. Once the team is happy with the model they've built and would like to progress the trained model from the TRE into a shared space that can be used in production, they will run a script that will create a copy of the model and dataset in a shared AML registry.
+The shared registry will then be accessed from the production environment to pull the model and dataset into the workspace or for model deployment.
+
+It is important to note that the AML registry is immutable
```

Comment:
Why is this important? It felt like the end of this sentence might have been missing.